generators/server/generator.mjs

```diff
@@ -17,6 +17,14 @@ export default class extends ServerGenerator {
 
   get [POST_WRITING_PRIORITY]() {
     return {
+      async increaseOauth2Sleep({ application: { applicationTypeOauth2 } }) {
+        if (applicationTypeOauth2) {
+          this.editFile(
+            'src/main/docker/app.yml',
+            content => content.replace('JHIPSTER_SLEEP=30', 'JHIPSTER_SLEEP=60')
```

Comment:
It doesn't seem like this is working because [the latest 21-points job shows](https://github.com/jhipster/generator-jhipster-ionic/runs/8178394620?check_suite_focus=true#step:18:186):

```
  The application will start in 30s...
```

Reply:
Yes, I need to check in a computer why.

Reply:
I think I found.